docs/getting_started.rst

```diff
@@ -53,9 +53,9 @@ other than complexity.
 If you write application code that is meant to run on various HPC clusters, but
 which never make calls to the underlying resource manager (e.g. by calling into
 Flux's client library, or executing ``srun``/``jsrun``/``aprun`` etc.), then
-PSI/J will not help you. This is likely your situation if you work on a MPI-based
-science simulation, since we have observed that it is often the users' responsibility
-to actually launch the simulation through the resource manager.
+PSI/J will not help you. This is likely your situation if you are a developer on a
```

Comment:
maybe "... if you are a developer working on..."?

Reply:
Works for me